tests/app/main/test_suppliers.py

```diff
@@ -1150,6 +1150,10 @@ def test_should_show_supplier_dashboard_logged_in(
             assert self.strip_all_whitespace("email@email.com") in \
                 self.strip_all_whitespace(res.get_data(as_text=True))
 
+            assert self.strip_all_whitespace("Change your password") in \
+                self.strip_all_whitespace(res.get_data(as_text=True))
+            assert "href=\"/user/change-password\"" in res.get_data(as_text=True)
+
```

Comment:
I'd... personally... be a little more stringent here...? But I guess it's not your job to convert all test assertions you find to xpath assertions.

Reply:
Yeah, we copied the format from a similar test elsewhere. Happy to change if there's a strong feeling (guessing not...)

Reply:
Might be nicer to use single-quotes on the outside as well, so you don't need to escape the double quotes in the HTML.

I agree with Rob that it feels weird to test the kink is present using this check. Xpath would probably be more sensible.